LoanReadForm.html

```diff
@@ -0,0 +1,20 @@
+$def with(edition_key)
+
+<style type="text/css">
+.darkgreen.local-date-time {
```

Comment:
Inline styles cause maintenance problems so we prefer to keep all CSS inside static/css

Please move these files to a file that is referenced in the entry point.
The entry point file you'll want to add these rules to is static/css/page-user.less

Reply:
Done